flight/targets/sparky2/fw/pios_board.c

```diff
@@ -154,30 +154,135 @@ static void panic(int32_t code) {
 void set_vtx_channel(HwSparky2VTX_ChOptions channel)
```

Comment:
Could you throw in a quick doxygen comment for this file? It's not immediately clear if you're not doing FPV that `vtx` is shorthand for "video transmission".